---
title: Jacqueline A. Robinson, PhD
---

## Research

I use whole genome sequencing data to study the evolutionary history and fitness of small isolated populations. Small populations face an elevated risk of extinction, in part due to genetic factors such as inbreeding depression, the accumulation of deleterious alleles over time, and reduced adaptive potential. I am particularly interested in exploring how patterns of genome-wide variation can illuminate the **evolutionary history** of populations and species, shed light on their **genetic health**, and inform **predictive models** to forecast population viability or extinction risk.

Jump to: [Vaquitas](#Vaquitas), [California condors](#CAcondors), [Baboons](#Baboons), [Isle Royale wolves](#IRwolves), [Channel Island foxes](#CIfoxes)

## <a name="Vaquitas"></a>Vaquitas

The vaquita porpoise (*Phocoena sinus*) is the world's smallest cetacean and, with ~10 surviving individuals, the world's most endangered marine mammal. Intense fishing pressure from gillnets used in the vaquita's sole habitat in the Gulf of California has led to a >99% decline of the species over the past century. Although these gillnets are used to target other species, vaquitas end up being caught and perishing as bycatch. Gillnet bans have been enacted to protect the vaquita, but are unfortunately they are not enforced, and the species cannot be brought into captivity for breeding and protection. Low genetic diversity in vaquitas resulting from their natural rarity has been cited in arguments that the species is doomed to extinction, and that conservation efforts should therefore be abandoned. To investigate this claim, we analyzed vaquita whole genome sequences and designed computational models to predict the species' extinction risk over the next 50 years. Although the vaquita population is now so small that future inbreeding is inevitable, our analyses show that there is a low risk of future inbreeding depression, and the species is highly likely to recover if gillnet mortality ceases immediately. Our study has implications for the conservation of vaquitas and other naturally rare species, and demonstrates how genomic data can be integrated into predictions of extinction risk.

**Robinson, J.A.**\*, Kyriazis, C.C.\*, Nigenda-Morales, S.F., Beichman, A.C., Rojas-Bracho, L., Robertson, K.M., Fontaine, M.C., Wayne, R.K., Lohmueller, K.E., Taylor, B.L., Morin, P.A., 2022. The critically endangered vaquita is not doomed to extinction by inbreeding depression. *Science* 376, 635-639. [doi:10.1126/science.abm1742](https://www.science.org/stoken/author-tokens/ST-482/full)

\*contributed equally

Featured coverage: [NY Times](https://www.nytimes.com/interactive/2022/05/05/climate/engandered-vaquitas-extinction-inbreeding.html), [BBC](https://www.bbc.com/news/science-environment-61326413), [NPR](https://www.npr.org/2022/05/05/1096842362/scientists-say-endangered-porpoise-isnt-doomed-so-long-as-humans-stop-killing-th), [Reuters](https://www.reuters.com/lifestyle/science/genome-study-offers-hope-diminutive-endangered-porpoise-2022-05-05/), [The Guardian](https://www.theguardian.com/environment/2022/may/05/vaquita-porpoise-could-survive-but-only-if-fishing-stops-immediately), [National Geographic](https://www.nationalgeographic.com/animals/article/vaquita-porpoises-could-genetically-recover-if-fishing-ends), [CBC: Quirks & Quarks](https://www.cbc.ca/listen/live-radio/1-51-quirks-and-quarks/clip/15911036-with-10-left-scientists-tiny-porpoise-survive), [Ars Technica](https://arstechnica.com/science/2022/05/humans-not-genetics-the-biggest-danger-for-the-vaquita/)

## <a name="CAcondors"></a>California condors

North America's largest flying bird, the California condor (*Gymnogyps californianus*), is an icon of the struggle to save species from extinction. California condors were pushed to the brink of extinction in the 20th century by a combination of human impacts including poaching, lead poisoning, and loss of nesting habitat. By 1982, only 22 individuals remained, and a captive breeding and release program was initiated to restore the species. These efforts have been greatly successful, and as of [December 2021](https://www.nps.gov/articles/000/caco-world-2021.htm) there are over 300 individuals in the wild and ~200 in captivity. To aid research and conservation of this species, we generated a high quality assembly of the California condor genome. We also examined genome-wide diversity to study the species' evolutionary history and its current status. Interestingly, California condor genomes retain a high degree of variation from their formerly large population sizes, on the order of tens of thousands of individuals, during the Pleistocene. However, the California condor genome also bears the hallmarks of inbreeding in the wild as the population declined to near-extinction. Future research will reveal whether some regions of the California condor genome have lost variability entirely, or if careful management can counteract the effects of the recent decline. The relatively high genome-wide diversity in California condors adds hope for their continued recovery.

**Robinson, J.A.**, Bowie, R.C.K., Dudchenko, O., Lieberman Aiden, E., Hendrickson, S.L., Steiner, C.C., Ryder, O.A., Mindell, D.P., and Wall, J.D., 2021. Genome-wide diversity in the California condor tracks its prehistoric abundance and decline. *Current Biology*. [doi:10.1016/j.cub.2021.04.035](https://doi.org/10.1016/j.cub.2021.04.035)

Featured coverage: [CBC: Quirks & Quarks](https://www.cbc.ca/radio/quirks/may-15-california-condor-genetics-a-strange-star-goes-supernova-don-t-think-just-kick-and-more-1.6025290/how-the-california-condor-escaped-extinction-and-a-genetic-bottleneck-1.6027240), [Berkeley News](https://news.berkeley.edu/2021/05/13/high-genomic-diversity-is-good-news-for-california-condor/), [The Wildlife Society](https://wildlife.org/recovering-wild-condors-maintain-high-genetic-diversity/), [Le Monde](https://www.lemonde.fr/sciences/article/2021/05/23/le-retour-gagnant-du-condor-de-californie_6081223_1650684.html), [The Academic Times](https://academictimes.com/the-california-condors-genome-encodes-a-hopeful-future-for-this-endangered-species/), [UCSF News Center](https://www.ucsf.edu/news/2021/08/421196/sequencing-condor-genome-insights-human-diversity)

## <a name="Baboons"></a>Baboons

Baboons (genus *Papio*) are social Old World primates that have been extensively studied by primatologists, ecologists, evolutionary biologists, and biomedical researchers. In the wild, baboons spend much of their time on the ground in savannas and open woodlands, making them ideal subjects for observational studies. This research has provided a wealth of information about the behavior and biology of baboons under natural conditions. In captivity, baboons are an important model for biomedical research because they share many similarities with humans relative to other model organisms, such as mice or fruit flies. I am studying whole genome sequences from a large pedigreed colony of olive and yellow baboons (*P. anubis* and *P. cynocephalus*) at the Southwest National Primate Research Center ([snprc.org](http://snprc.org/)) to investigate mutation and recombination across the  baboon genome. Our genomic dataset allows us to study these processes in ways that are extremely challenging to do in wild populations, and have therefore mostly been done in a small number of model species. Our research aims to expand our knowledge of these evolutionary forces in a species that shares many similarities with humans.

**Robinson, J.A.**, Birnbaum, S., Newman, D.E., Chan, J., Glenn, J.P., Cox, L.A. and Wall, J.D., 2019. Analysis of 100 high coverage genomes from a pedigreed captive baboon colony. *Genome Research* 29, 848-856. [doi:10.1101/gr.247122.118](https://doi.org/10.1101/gr.247122.118)

Batra, S.S., Levy-Sakin, M., **Robinson, J.**, Guillory, J., Durinck, S., Vilgalys, T.P., Kwok, P.Y., Cox, L.A., Seshagiri, S., Song, Y.S., and Wall, J.D., 2020. Accurate assembly of the olive baboon (Papio anubis) genome using long-read and Hi-C data. *GigaScience* 9, giaa134. [doi:10.1093/gigascience/giaa134](https://doi.org/10.1093/gigascience/giaa134)

Wall, J., **Robinson, J.A.**, Cox, L.A., 2022. High-resolution estimates of crossover and noncrossover recombination from a captive baboon colony. *Genome Biology and Evolution* 14, evac040. [doi:10.1093/gbe/evac040](https://doi.org/10.1093/gbe/evac040)

## <a name="IRwolves"></a>Isle Royale wolves

For more than 50 years, biologists have been studying gray wolves (*Canis lupus*) and moose (*Alces alces*) on Isle Royale in Lake Superior as part of the longest-running study of predator-prey interactions in the wild. Over that time, biologists witnessed the rise of the island's wolf population to a peak of 50 individuals, before it declined to just two individuals and became severely inbred. Over time, inbreeding within the isolated wolf population led to high rates of skeletal defects and an end to successful reproduction. In collaboration with Isle Royale biologists and an expert on wolf skeletal morphology, we examined the genomic consequences of severe inbreeding depression in Isle Royale wolves. Our study sheds light on the effects of recent inbreeding versus long-term limited population size, and suggests that the number of wolves on Isle Royale is simply too low to sustain a viable population in the long run without an influx of new individuals to counteract inbreeding. These results have implications for the translocation of wolves to Isle Royale that began in 2018, and for the management of other extremely small and isolated populations.

**Robinson, J.A.**, Räikkönen, J., Vucetich, L.M., Vucetich, J.A., Peterson, R.O., Lohmueller, K.E. and Wayne, R.K., 2019. Genomic signatures of extensive inbreeding in Isle Royale wolves, a population on the threshold of extinction. *Science Advances* 5, eaau0757. [doi:10.1126/sciadv.aau0757](https://doi.org/10.1126/sciadv.aau0757)

Hedrick, P.W., **Robinson, J.A.**, Peterson, R.O. and Vucetich, J.A., 2019. Genetics and extinction and the example of Isle Royale wolves. *Animal Conservation* 22, 302–309. [doi:10.1111/acv.12479](https://doi.org/10.1111/acv.12479)

Featured coverage: [AAAS](https://www.aaas.org/news/inbreeding-causes-isle-royale-wolf-population-crash)

## <a name="CIfoxes"></a>Channel Island foxes

Populations that are small and isolated face an elevated risk of extinction, partly due to genetic factors. Experiments in the lab have repeatedly demonstrated that small and isolated populations are doomed to mutation accumulation and, ultimately, extinction, but it remains unclear whether this poses a significant risk of extinction in the wild. In the 1990s, several studies revealed that island foxes (*Urocyon littoralis*) on California's Channel Islands harbor exceptionally low levels of genetic diversity, indicating a long history of small population size and isolation. In particular, foxes on San Nicolas Island were found to be genetically indistinguishable from one another. This level of genetic monomorphism has not been observed in any other wild population, and begs the question of how island foxes have persisted for thousands of years without showing obvious physical symptoms of genetic deterioration. In two publications, we present the results of whole genome sequence analyses, morphological examinations, and simulations of island fox evolution to understand this apparent paradox. Our studies suggest island fox genomes have slightly more weakly deleterious mutations, but fewer strongly deleterious recessive mutations, enabling them to thrive in their island habitats without suffering from severe inbreeding depression.

**Robinson, J.A.**, Ortega Del-Vecchyo, D., Fan, Z., Kim, B.Y., vonHoldt, B.M., Marsden, C.D., Lohmueller, K.E., and Wayne, R.K., 2016. Genomic flatlining in the endangered island fox. *Current Biology* 26, 1183–1189. [doi:10.1016/j.cub.2016.02.062](https://doi.org/10.1016/j.cub.2016.02.062)

**Robinson, J.A.**, Brown, C., Kim, B.Y., Lohmueller, K.E., and Wayne, R.K., 2018. Purging of strongly deleterious mutations explains long-term persistence and absence of inbreeding depression in island foxes. *Current Biology* 28, 3487–3494. [doi:10.1016/j.cub.2018.08.066](https://doi.org/10.1016/j.cub.2018.08.066)

Featured coverage: [NY Times](https://www.nytimes.com/2016/04/26/science/channel-island-foxes-least-genetic-diversity.html), [Scientific American](http://blogs.scientificamerican.com/extinction-countdown/island-fox-genetics), [American Scientist](https://www.americanscientist.org/article/the-fates-of-channel-island-foxes-and-isle-royale-wolves), [Why Evolution Is True](https://whyevolutionistrue.wordpress.com/2016/04/28/tiny-foxes-on-the-channel-islands-lack-not-only-mass-but-also-genetic-variation-and-fear-of-humans), [The Science Explorer](http://thescienceexplorer.com/nature/tiny-island-foxes-are-treading-shallow-gene-pool), The Molecular Ecologist ([1](https://www.molecularecologist.com/2016/05/what-does-the-island-fox-say/), [2](https://www.molecularecologist.com/2018/11/how-island-foxes-are-living-on-the-edge/))

